src/main/java/org/apache/mesos/scheduler/plan/DefaultBlock.java

```diff
@@ -17,24 +20,23 @@
     private final String name;
     private final Optional<OfferRequirement> offerRequirementOptional;
     private final UUID id = UUID.randomUUID();
+    private final List<String> errors;
+    private final Strategy strategy = new ParallelStrategy();
     private Status status;
     private Map<Protos.TaskID, Status> tasks = new HashMap<>();
 
-    public DefaultBlock(String name, OfferRequirement offerRequirement, Status status) {
-        this(name, Optional.of(offerRequirement), status);
-    }
-
-    private DefaultBlock(String name, Optional<OfferRequirement> offerRequirementOptional, Status status) {
+    public DefaultBlock(
+            String name,
+            Optional<OfferRequirement> offerRequirement,
+            Status status,
+            List<String> errors) {
         this.name = name;
-        this.offerRequirementOptional = offerRequirementOptional;
+        this.offerRequirementOptional = offerRequirement;
```

Comment:
May as well drop the 'optional' on the internal value's name as well (dont care which way this goes, as long as the way is consistent for the file)